src/types/images.d.ts

```diff
@@ -0,0 +1,5 @@
+// These are to use import images without TS module not found error
```

Comment:
It looks like this is outputting an empty  `images.d.js` file. Did you see the same?

Reply:
ok a lot of trial error here. I moved `types` to root so it doesnt output in `lib` had to include it in the tsconfig manually, thankfully that doesnt . make it generate an output from types folder